scripts/entrypoint.pl

```diff
@@ -134,21 +134,26 @@ sub cmd_checksetup {
 
 sub cmd_load_test_data {
   wait_for_db();
-
-  die 'BZ_ANSWERS_FILE is not set' unless $ENV{BZ_ANSWERS_FILE};
-  run('perl', 'checksetup.pl', '--no-template', $ENV{BZ_ANSWERS_FILE});
-
-  run(
-    'perl',        'scripts/generate_bmo_data.pl',
-    '--param',     'use_mailer_queue=0'
-  );
-
-  if ($ENV{BZ_QA_CONFIG}) {
-    chdir '/app/qa/config';
-    say 'chdir(/app/qa/config)';
-    run('perl', 'generate_test_data.pl');
-    chdir '/app';
-  }
+  use Capture::Tiny qw(capture_merged);
+
+  say 'Loading test data...';
+
+  my $output = capture_merged {
+    die 'BZ_ANSWERS_FILE is not set' unless $ENV{BZ_ANSWERS_FILE};
+    run('perl', 'checksetup.pl', '--no-template', $ENV{BZ_ANSWERS_FILE});
+
+    run(
+      'perl',        'scripts/generate_bmo_data.pl',
+      '--param',     'use_mailer_queue=0'
+    );
+
+    if ($ENV{BZ_QA_CONFIG}) {
+      chdir '/app/qa/config';
+      say 'chdir(/app/qa/config)';
+      run('perl', 'generate_test_data.pl');
+      chdir '/app';
+    }
+  };
```

Comment:
Capturing and dropping the output when there's an error makes it impossible to diagnose issues; it should be displayed on failure.

Running `test_selenium` results in the following error for me:

```
Loading test data...
"perl" unexpectedly returned exit value 255 at /app/local/lib/perl5/Capture/Tiny.pm line 382.
 at /app/scripts/entrypoint.pl line 304
```

Commenting out `capture_merged` revealed:

```
Loading test data...
+ perl checksetup.pl --no-template /app/conf/checksetup_answers.txt
Creating /app/data/SiteMapIndex directory...
Creating /app/data/assets directory...
Creating /app/data/attachments directory...
Creating /app/data/db directory...
Creating /app/data/extensions directory...
Creating /app/data/mining directory...
Creating /app/data/webdot directory...
Creating /app/data/extensions/additional...
Creating /app/data/mailer.testfile...
Creating __lbheartbeat__...
Creating js/yui.js...
Creating js/yui3.js...
Creating skins/yui.css...
Creating skins/yui3.css...
Creating version.json...
Deriving regex group memberships...
+ perl scripts/generate_bmo_data.pl --param use_mailer_queue=0
creating user accounts...
creating classifications...
creating products...
creating groups...
creating custom fields
creating flag types...
creating keywords...
installation and configuration complete!
chdir(/app/qa/config)
+ perl generate_test_data.pl
reading the config file...
The value 'Highest' already exists for the Priority field.
"perl" unexpectedly returned exit value 255 at /app/local/lib/perl5/IPC/System/Simple.pm line 578.
	IPC::System::Simple::_check_exit("perl", 255, ARRAY(0x561cfdee9fc0)) called at /app/local/lib/perl5/IPC/System/Simple.pm line 550
	IPC::System::Simple::_process_child_error(65280, "perl", ARRAY(0x561cfdee9fc0)) called at /app/local/lib/perl5/IPC/System/Simple.pm line 217
	IPC::System::Simple::runx(ARRAY(0x561cfdee9fc0), "perl", "generate_test_data.pl") called at /app/local/lib/perl5/IPC/System/Simple.pm line 168
	IPC::System::Simple::run("perl", "generate_test_data.pl") called at (eval 416) line 12
	eval {...} called at (eval 416) line 11
	Fatal::__ANON__("perl", "generate_test_data.pl") called at /app/scripts/entrypoint.pl line 304
	main::run("perl", "generate_test_data.pl") called at /app/scripts/entrypoint.pl line 153
	main::cmd_load_test_data() called at /app/scripts/entrypoint.pl line 180
	main::cmd_test_qa("test_*.t") called at /app/scripts/entrypoint.pl line 173
	main::cmd_test_selenium() called at /app/scripts/entrypoint.pl line 70
 at /app/scripts/entrypoint.pl line 304
```